File: PALL01.py
t = input()
while(t > 0):
	reverse = 0
	t -= 1
	n = input()
	num = n
	while(n != 0):
		reverse = reverse*10 + n%10
		n /= 10
	if(num == reverse):
		print 'wins'
	else:
		print 'losses'

File: chef/PALL01.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
	int t;
	cin >> t;
	while(t--)
	{
		long long int n,num,rev=0;
		cin >> num;
		n = num;
		while(n > 0)
		{
			rev = rev*10 + n%10;
			n = n/10;
		}
		if(num == rev)
			cout << "wins\n";
		else
			cout << "losses\n";
	}
	return 0;
}